sixMans/sixMans.py

```diff
@@ -678,7 +756,7 @@ async def monthly(self, ctx: Context, player: discord.Member = None, *, queue_na
 
     #endregion
 
-    #region get and set commands
+#region get and set commands
```

Comment:
Did you mean to shift this over? Looks like the other sub-regions inside the command region are tabbed one to the right to make it more obvious that it's within the overall command region

Reply:
Not sure but it can be undone

Reply:
Yes, this was intentional because I was able to further organize the code structure with these comment indents. These indented comments allowed me to collapse additional sections of code, which I used to divide player and admin commands.
![image](https://user-images.githubusercontent.com/16944592/122508120-6c794980-cfcf-11eb-8d9d-144d07f10a8c.png)

Reply:
I just realized that I made a mistake with one of these comments. The indentation works great but I had a closing group comment as `#regionend` instead of `endregion` which broke the behavior. This has been resolved.
![image](https://user-images.githubusercontent.com/16944592/122591172-957df680-d030-11eb-9b6f-0c24badbf659.png)

Reply:
Though I may have broke organization in other ways? We can make more changes if we see its fit.